### ISSUE TITLE
Make `ChoosePathString` accessible from GDScript

### ISSUE THREAD

**Issue by greenfox1505** [paulloz/godot-ink#53]
**Describe the bug**
```gdscript
var story = load("res://story.ink")
story.ChoosePathString("main",true)
story.ChoosePathString("main",true,[])
```
This function doesn't expose to GDScript.

**To Reproduce**
load a story in gdscript, try to access that function

**Environment**
* OS: Windows
* Godot version: 4.1.rc1.mono
* godot-ink version: 1.0.0
* ink version: any? I think?

**Additional context**
```cs
//in InkStory.cs, add this
	public void ChoosePathString(string path, bool resetCallstack)
	{
		runtimeStory.ChoosePathString(path, resetCallstack);
	}

	public void ChoosePathStringArgs(string path, bool resetCallstack, Variant[] arguments)
	{
		runtimeStory.ChoosePathString(path, resetCallstack,FromVariants(arguments));
	}
```
GDScript doesn't know how to bind against C# `params`, so that function isn't usable. Removing the `params` fixes it, but causes a collision and would break compatibility. I added this function and this new signature to retain compatibility and allow that functality to be accessible to GDScript.

**Comment by wjagodfrey** [comment]
Just bumped into this. Sad times. @paulloz please add this fix

**Comment by greenfox1505** [comment]
If you throw my fix here into your repo, it should work for now.

**Comment by paulloz** [comment]
Hello. There's usually little need to bump issues up once they've been reported and acknowledged. This binding issue will be fixed properly when I'm able to spend time on it.

**Comment by paulloz** [comment]
This is now fixed on `main`, and will be included in the next release (`v1.1.0`).
